Project.toml

```diff
@@ -8,11 +8,12 @@ version = "0.5.2"
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 DensityInterface = "b429d917-457f-4dbc-8f4c-0cc954292b1d"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
```

Comment:
We don't want to depend on Distributions in the main package. Rather, add it as a dependency to the Project.toml in test/.